### PR TITLE
[Feature] Deploy swedish admin

### PR DESCRIPTION
**Pull request by lorentzlasson**
Triggering two parallel deploys for SE and NO when pushed to master.

Haven't tested the full deployment flow, but tested the swedish subset [here](https://console.cloud.google.com/cloud-build/builds;region=global/acdb2105-95c4-4b84-aef5-f6fa07d93f65?project=hidden-display-243419). Would like to try out the dev flow before merging

#355 

---

Tested on devices 
- [ ] Desktop 💻
- [ ] Mobile 📱

Tests  
- [ ] All tests are running ✔️
- [ ] Test are updated 🧪
- [ ] Code Review 👩‍💻
- [ ] QA 👌

⏲️ Time spent on CR:

⏲️ Time spent on QA:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,4 +22,4 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 .eslintcache
-src/config/config.ts
```
Comment (lorentzlasson) [inline .gitignore:25]: It's checked in to the repo so figured we shouldn't pretend it's ignored

File: .gitignore [hunk 2]
```diff
@@ -22,4 +22,4 @@ npm-debug.log*
...
+.env
```
Comment (lorentzlasson) [inline .gitignore:25]: Continuing with the .env.example -> .env workflow

File: .env.example [hunk 3]
```diff
@@ -0,0 +1,2 @@
+HOST=localhost
```
Comment (lorentzlasson) [inline .env.example:1]: Is this used for anything? AFAIK know `REACT_APP_*` envs are let through

File: clouddeploy.yml [hunk 4]
```diff
@@ -1,11 +1,31 @@
...
   env: ['CI=true']
   args: ['run', 'test']
-- name: 'gcr.io/cloud-builders/npm'
-  env: ['CI=true']
+
+- id: 'build-norway'
+  name: 'gcr.io/cloud-builders/npm'
+  env: ['CI=true', 'REACT_APP_API_URL=https://data.gieffektivt.no']
+  args: ['run', 'build']
+  waitFor: ['test']
```
Comment (lorentzlasson) [inline clouddeploy.yml:15]: Should run the build & deploys in parallel at least

**Review by adamalfredsson — APPROVED** [review]
Yay! Looks good! I'm not sure about reviewing the cloud build config

File: cloudbuild.yml [hunk 5]
```diff
@@ -1,9 +1,13 @@
 steps:
-- name: 'gcr.io/cloud-builders/npm'
+- id: 'install'
+  name: 'gcr.io/cloud-builders/npm'
   args: ['install']
-- name: 'gcr.io/cloud-builders/npm'
+
+- id: 'type-check'
+  name: 'gcr.io/cloud-builders/npm'
+  args: ['run', 'typecheck']
```
[the remark this answers is not in the document]
Comment (fellmirr) [reply cloudbuild.yml:8]: :+1:

File: .env.example (hunk 3 above)
Comment (fellmirr) [reply .env.example:1]: I have no idea? 🤷 Global search for `process.env.HOST` or similar should yield results if it's used.

File: .gitignore (hunk 1 above)
Comment (fellmirr) [reply .gitignore:25]: I'm assuming it's ignored because it acted as an env-ish file, but I agree, let's not do this from now :+1:

**Pull request merged**